package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@viaprotocol/web3-wallets",
-  "version": "1.0.15",
+  "version": "1.0.16",
```

Comment:
Please, upd version only when task in `master` branch. Might be conflicts plus need release commit anyway

Reply:
Ok, sorry. Removed.